ftw/upgrade/tests/test_command_create.py

```diff
@@ -52,6 +52,27 @@ def test_creating_an_upgrade_step_with_specifying_upgrades_directory(self):
             'Expected subpackage upgrades directory to have one upgrade; {0}'.format(
                 subpackage_upgrades_dir.listdir()))
 
+    def test_creating_an_upgrade_step_with_text_containing_dots(self):
+        package = create(Builder('python package')
+                         .named('the.package')
+                         .at_path(self.layer.sample_buildout)
+                         .with_directory('upgrades'))
+
+        self.upgrade_script('create "Update ftw.upgrade to version 3."')
+
+        upgrades_dir = package.package_path.joinpath('upgrades')
+        step_path, = upgrades_dir.listdir()
+        self.assertRegexpMatches(step_path.name,
+                                 r'^\d{14}_update_ftw_upgrade_to_version_3$')
+
+        code_path = step_path.joinpath('upgrade.py')
+        self.assertTrue(code_path.exists(), 'upgrade.py is missing')
+        self.maxDiff = True
+        self.assertIn('class UpdateFtwUpgradeToVersion3(UpgradeStep):',
+                      code_path.text())
+        self.assertIn("""Update ftw upgrade to version 3.\n    """,
```

Comment:
As discussed: we could determine of the user enters a camel cased sentence or a human readable sentence and convert accordingly or use the sentence as is for the docstring.